content/docs/pulumi-cloud/get-started/onboarding-guide/migrating-to-pulumi.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Migrating to Pulumi
+meta_desc: Focus on building momentum with early wins while establishing practices that will scale with your organization.
+weight: 4
+menu:
+    cloud:
+        name: Migrating to Pulumi
+        parent: pulumi-onboarding-guide
+        identifier: migrating-to-pulumi
+---
+Successfully migrating to Pulumi requires strategic decisions about your migration approach. Focus on building momentum with early wins while establishing practices that will scale with your organization.
+
+## Migrating existing infrastructure
+
+If you have existing cloud infrastructure to bring into Pulumi IaC, you have several strategies available.
+
+### Choose your migration approach
+
+**Start fresh:** Simply throw away existing infrastructure and begin anew. This ensures you can adopt all best practices from the outset without technical debt. This option isn't always practical for business-critical services.
+
+**Import existing infrastructure:** Pulumi has tools to import any cloud infrastructure regardless of how it was created — even manually through cloud consoles. Pulumi offers tailored migration tools for Terraform, AWS CloudFormation/CDK, Azure ARM, and Kubernetes YAML. These tools generate Pulumi IaC code in your chosen language and actively place existing resource management under Pulumi IaC, swapping out management without disrupting resources for zero downtime.
```

Comment:
Add a link and recommended approach language to using Insights visual import
https://www.pulumi.com/docs/insights/import/